packages/knip/src/plugins/lefthook/index.ts

```diff
@@ -54,6 +50,10 @@ export default {
   title,
   enablers,
   isEnabled,
-  config,
+  get config() {
+    const gitHookPaths = getGitHookPaths();
+
+    return ['lefthook.yml', ...gitHookPaths];
+  },
```

Comment:
This is a workaround for configs getting calculated way too early with a static `process.cwd()